lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -1008,12 +992,12 @@ struct ConvertLayoutOpConversion
                              const DotOperandEncodingAttr &dotOperandLayout,
                              bool isOuter) const {
     auto loc = op.getLoc();
-    Value src = op.getSrc();
-    Value dst = op.getResult();
+    auto src = op.getSrc();
```

Comment:
maybe we should talk about `auto` uses as it sounds like different people have different preferred usages and it would be nice to not changes things back and forth